src/init.lua

```diff
@@ -1045,9 +1045,9 @@ xplr.config.modes.builtin.default = {
         help = "global help menu",
         messages = {
           {
-            BashExec = [===[
+            BashExec0 = [===[
               [ -z "$PAGER" ] && PAGER="less -+F"
-              cat -- "${XPLR_PIPE_GLOBAL_HELP_MENU_OUT}" | ${PAGER:?}
+              cat -- "${XPLR_PIPE_GLOBAL_HELP_MENU_OUT}" | tr '\0' '\n' | ${PAGER:?}
```

Comment:
Wouldn't this be a case where `BashExec` would be a better fit?

Reply:
Right, but I think it's better to stay consistent.

Reply:
Is the idea that the non-null-handling versions will eventually be deprecated? If not, why wouldn't you prefer the best tool for the job?

Reply:
They won't be deprecated, just not recommended. I have no argument against using BashExec here other than

- BashExec0 is consistent throughout the init.lua file.
- Works as an example of how to handle `\0` using `tr`.

But they are weak arguments, so I'll think more on this. Please feel free to provide more suggestions though.

Reply:
I hope you don't take my criticisms too personally. I know I have this tendency. I'm really just curious about the reasoning, and you obviously know the codebase much better than I do.

Reply:
Not at all, I'm glad you're raising them. your points are valid. We all have opinions.